src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelProxyExecutionManager.cs

```diff
@@ -22,6 +24,8 @@ internal class ParallelProxyExecutionManager : ParallelOperationManager<IProxyEx
         #region TestRunSpecificData
 
         private int runCompletedClients = 0;
+        private int runStartedClients = 0;
```

Comment:
Can you please explain why we need this variable? Why runCompletedClients is not fulfilling our requirement?